.github/workflows/build.yml

```
@@ -0,0 +1,30 @@
+        run: git fetch --unshallow --prune
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+            dotnet-version: 9.0.x            
+      - name: Build Cake.CycloneDX
+        uses: cake-build/cake-action@v3
+        with:
+          project-path: build/Build.csproj
+          target: All
+          verbosity: Diagnostic
+          
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)